lib/ansible/cli/vault.py

```diff
@@ -384,7 +384,7 @@ def execute_encrypt_string(self):
                 sys.stderr.write(err)
             b_outs.append(to_bytes(out))
 
-        self.editor.write_data(b'\n'.join(b_outs), context.CLIARGS['output_file'] or '-')
```

Comment:
It would a good idea to add a comment before the line, to prevent accidental removal in the future. `# Output must end with a newline to prevent bug #78932`

Reply:
@ssbarnea Done